benchexec/tablegenerator/react-table/src/utils/utils.js

```diff
@@ -210,7 +210,7 @@ const EXTENDED_DISCRETE_COLOR_RANGE = [
  */
 const getURLParameters = (str) => {
   // Split the URL string into parts using "?" as a delimiter
-  const urlParts = (str || decodeURI(document.location.href)).split("?");
+  const urlParts = (str || document.location.href).split("?");
```

Comment:
I guess this function should get some comment explaining that it handles decoding of the parameters, except for `filter` where it does not do that on purpose to allow the filter handling do it on its own?